docs/Remix.md

```diff
@@ -321,18 +367,31 @@ Now, start the server with `npm run dev`, browse to `http://localhost:3000/`, an
 
 In many cases, the admin is only a part of the application. For instance, you may want to render the admin in a subpath like `/admin`.
 
-To do so, add a [splat route](https://remix.run/docs/en/main/file-conventions/route-files-v2#splat-routes), i.e. a route that matches all URLs inside a sub path. A splat route is named `$.tsx`. Create a file called `app/routes/admin.$.tsx` file with the following content:
+To do so, add a [splat route](https://remix.run/docs/en/main/file-conventions/routes#splat-routes), i.e. a route that matches all URLs inside a sub path. A splat route is named `$.tsx`. Create a file called `app/routes/admin.$.tsx` file with the following content:
 
 ```jsx
 // in app/routes/admin.$.tsx
-import App from "../components/App";
+import { Admin, Resource, ListGuesser } from "react-admin";
+import jsonServerProvider from "ra-data-json-server";
 import styles from "~/styles/app.css";
 
 export function links() {
   return [{ rel: "stylesheet", href: styles }];
 }
 
-export default App;
+const dataProvider = jsonServerProvider.default
+  ? jsonServerProvider.default("https://jsonplaceholder.typicode.com")
+  : jsonServerProvider("https://jsonplaceholder.typicode.com");
+
+export default function App() {
+  return (
+    <Admin basename="/admin" dataProvider={dataProvider}>
+      <Resource name="posts" list={ListGuesser} />
+      <Resource name="comments" list={ListGuesser} />
+    </Admin>
+  );
+}
```

Comment:
Why not reuse the `App` component here? Following the tutorial, we now have a useless `App.tsx` file...

Besides, this file is still referenced in the following paragraph, so you should definitely keep it!

Reply:
In fact, I think we should remove the "Using React-Admin As The Root Application" altogether